lib/restforce/collection.rb

```diff
@@ -33,6 +33,12 @@ def size
     end
     alias length size
 
+    def count
```

Comment:
Do we need to do something to handle the `count(item) -> int` 
case? See the Ruby standard library docs [here](https://ruby-doc.org/core-3.0.2/Enumerable.html#method-i-count).

Reply:
I'm wondering if it should be `def count(item = nil)`.

Reply:
Yes, good catch!

`item = nil` wouldn't work, it would count how many items would equal to `nil`. Declaring it to take any number of parameters works as `super` forwards the arg list as is it then allows the C land to inspect it. Will look a bit ugly in the docs though :/

Reply:
I don't write much Ruby anymore so I'm glad I've still got it in me - I did slightly fear my feedback could be totally off base 😅